### PR TITLE
Observation list and layer list UI styling

### PR DESCRIPTION
**Pull request by gyarill**

### CODE REVIEW

**Comment by gyarill** [comment]
Screen Shot 2020-03-18 at 5 57 51 PM

File: web-ng/src/app/components/map/ground-pin.ts [hunk 1]
```diff
@@ -14,8 +14,14 @@
...
-export const renderPin = (color: string) => {
-  return svgTemplate.replace('{{ color }}', color);
+export const defaultIconColor = 'red';
+
+export const renderPin = (color: string | undefined) => {
+  return svgTemplate.replace('{{ color }}', color || defaultIconColor);
+};
+
+export const getPinSrc = (color: string | undefined) => {
+  return '/assets/img/pin-red.svg';
```
Comment (gino-m) [inline web-ng/src/app/components/map/ground-pin.ts]: Can you we renderPin() here instead of having a static marker?

Comment (gyarill) [reply web-ng/src/app/components/map/ground-pin.ts]: done

File: web-ng/src/app/components/map/map.component.ts [hunk 2]
```diff
@@ -83,11 +83,9 @@ export class MapComponent implements OnInit {
     // Consider break this down when more inputs are available for <map-marker>.
     const normalScale = 30;
     const enlargedScale = 50;
-    const defaultIconColor = 'red';
-    const color =
-      project.layers.get(feature.layerId)?.color || defaultIconColor;
+    const color = project.layers.get(feature.layerId)?.color;
     const icon = {
-      url: 'data:image/svg+xml;charset=UTF-8;base64,' + btoa(renderPin(color)),
+      url: getPinDangerousSrc(color),
```
Comment (gino-m) [inline web-ng/src/app/components/map/map.component.ts]: Why DangerousSrc?

**Comment by gino-m** [comment]
@DaoyuT could you PTAL?

File: web-ng/src/app/components/map/map.component.ts (hunk 2 above)
Comment (DaoyuT) [reply web-ng/src/app/components/map/map.component.ts]: +1

**Pull request merged**